tests/suites/test_suite_pk.pk_can_do_psa.data

```diff
@@ -0,0 +1,260 @@
+PK can do PSA: input data validation
+depends_on:PSA_HAVE_ALG_SOME_ECDSA:!MBEDTLS_ECP_DP_SECP224K1_ENABLED
+pk_can_do_psa_fail:
+
+PK can do PSA: MBEDTLS_PK_ECDSA, check sign hash - Success
+depends_on:PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY:PSA_HAVE_ALG_SOME_ECDSA:!MBEDTLS_ECP_DP_SECP224K1_ENABLED
+pk_can_do_psa:MBEDTLS_PK_ECDSA:PSA_KEY_TYPE_ECC_KEY_PAIR(MBEDTLS_TEST_PSA_ECC_ONE_FAMILY):PSA_KEY_USAGE_EXPORT:MBEDTLS_PK_ALG_ECDSA(PSA_ALG_ANY_HASH):PSA_ALG_NONE:MBEDTLS_TEST_PSA_ECC_ONE_CURVE_BITS:MBEDTLS_PK_ALG_ECDSA(PSA_ALG_SHA_256):PSA_KEY_USAGE_SIGN_HASH:1
+
```

Comment:
Missing coverage: `key_alg = f(h1)` and `alg_check = f(h2)` where `h1 ≠ PSA_ALG_ANY_HASH` and `h1 ≠ h2`.